src/index.html

```diff
@@ -5,8 +5,275 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap&subset=cyrillic,cyrillic-ext,latin-ext" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+  <header class="header">
+    <a href="#" class="header__logo"><img src="images/logo.png" alt="Moyo logo"></a>
+    <nav class="menu">
+      <ul class="menu__list">
+        <li class="menu__list-item">
+          <a href="#" class="menu__link is-active">Apple</a>
+        </li>
+        <li class="menu__list-item">
+          <a href="#" class="menu__link">Samsung</a>
+        </li>
+        <li class="menu__list-item">
+          <a href="#" class="menu__link">Смартфоны и Телефоны</a>
+        </li>
+        <li class="menu__list-item">
+          <a href="#"  class="menu__link" data-qa="nav-hover">Ноутбуки И Компьютеры</a>
+        </li>
+        <li class="menu__list-item">
+          <a href="#" class="menu__link">Гаджеты</a>
+        </li>
+        <li class="menu__list-item">
+          <a href="#" class="menu__link">Планшеты</a>
+        </li>
+        <li class="menu__list-item">
+          <a href="#" class="menu__link">Фото</a>
+        </li>
+        <li class="menu__list-item">
+          <a href="#" class="menu__link">Видео</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main class="product-wrapper">
+      <section class="product" data-qa="card">
+
+        <a href="">
+          <img class="product__img" src="images/imac.jpeg" alt="imac">
+        </a>
+
+        <div class="product__list">
```

Comment:
`...list` де тут є список?
```suggestion
        <div class="product__info">
```